server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
+    doc: 'The url of the Firefox Sync tokenserver',
+    env: 'SYNC_TOKENSERVER_URL',
```

Comment:
> would we know right away if there was an uptick in the number of Sync connection errors?

No. This being wrong would be very bad. TPS would give us *some* visibility, but it would still be very bad.

> If we do encounter a connection error because of a misconfigured URL or the Sync token server just being down, what does the browser do to recover?

This will look like a non-auth, network related error and retry forever. Which is a really good point - if we don't re-fetch the config in this case we are stuck - in the wrong place in the desktop code to be telling FxA to re-configure itself.

While I like the idea of a single pref, the fact the token-server config is thorny makes me concede we should probably drop it from here for now.